Gopkg.toml

```diff
@@ -26,6 +26,21 @@
 
 
 
+required = [
+  "github.com/Azure/go-autorest/autorest",
+	"github.com/docker/distribution/reference"
+]
+
+
+
+[[constraint]]
+  name = "github.com/Azure/go-autorest"
+  version = "9.1.0"
```

Comment:
Is that needed?

Reply:
Well yes, which is why I added it. At least I did not find a way to get the deps compiled without it. These things are always because of Kubernetes dpendencies. When you find a nicer more default way to compile this shit, let me know.